docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+# GeoNetwork
```

Comment:
@tianon @yosifkit Is there any best practice concerning having docker-compose files in an official docker image repository?

Reply:
You are welcome to add any to your repo, but they won't really be tracked by us. We do have support for adding a single example `stack.yml` to the [Docs repo](https://github.com/docker-library/docs/) (and you already have [one](https://github.com/docker-library/docs/blob/e4dc26fea4ff449f53837daed1f348363c072423/geonetwork/stack.yml)) and you are welcome to link to others or embed them in the `content.md`.

Reply:
@yosifkit Thanks, I have merged this and added a PR in the library [here](https://github.com/docker-library/official-images/pull/5347).